### PR TITLE
Add availability keys to doc output.

### PR DESCRIPTION
**Pull request by jverkoey**
Includes:

- key.always_deprecated
- key.always_unavailable
- key.deprecation_message
- key.unavailable_message

For example, always_deprecated is true if an API has the `__attribute__((deprecated))` attribute. deprecation_message will include the message if one was provided.

We will be using this in the https://github.com/material-motion/material-motion-apidiff repo to generate deprecation notices in our release notes.

Closes https://github.com/material-motion/material-motion-apidiff/issues/10
Related to https://github.com/jpsim/SourceKitten/issues/214

### CODE REVIEW

**Comment by jverkoey** [comment]
This PR is blocked by https://github.com/jpsim/SourceKitten/pull/245.

**Comment by jverkoey** [comment]
I'm guessing tests are failing due to presence of new keys. I'm assuming there exist some tests that have cached generated output that is being compared against the live generated output.

**Comment by jverkoey** [comment]
Xcode unit tests pass. Will try running `make test` to see if I can repro failure locally.

**Comment by jverkoey** [comment]
Updated the fixtures.

**Comment by jpsim** [comment]
This is super great @jverkoey! 👏 

Could you please add a changelog entry for this marvelous update?

**Comment by jverkoey** [comment]
Donezo :)

**Comment by jpsim** [comment]
👏 many thanks. I really like this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,12 +25,13 @@
...
-* The following availability and deprecation values are now exposed for Objective-C APIs.  
-  ([Jeff Verkoeyen](https://github.com/jverkoey))
+* The following availability and deprecation values are now exposed for Objective-C APIs.
+
   * key.always_deprecated
   * key.always_unavailable
   * key.deprecation_message
-  * key.unavailable_message
+  * key.unavailable_message  
+  [Jeff Verkoeyen](https://github.com/jverkoey)
```
Comment (jverkoey) [inline CHANGELOG.md]: Thanks! Wasn't sure how best to format this.

**Comment by jverkoey** [comment]
Stoked to use this to resolve https://github.com/material-motion/material-motion-apidiff/issues/10.

**Comment by jpsim** [comment]
Very cool!

**Pull request merged**